src/chains/ethereum/ethereum/src/forking/state-manager.ts

```diff
@@ -42,7 +43,7 @@ export class ForkStateManager extends GanacheStateManager {
    * at the last fully committed point, i.e. as if all current
    * checkpoints were reverted.
    */
-  copy(): GanacheStateManager {
+  copy(): ForkStateManager {
```

Comment:
Another one that I changed that's unrelated to the merge. But figured the types should line up and I was here so...